tests/roles/pull_openstack_configuration/tasks/os_diff_pull_config.yaml

```diff
@@ -28,28 +28,43 @@
 - name: Set ssh command for Tripleo standalone in os-diff.cfg
   ansible.builtin.lineinfile:
     path: "{{ os_diff_dir }}/os-diff.cfg"
-    regex: "ssh -F ssh.config standalone"
-    value: "ssh_cmd=ssh -i {{ standalone_private_key | default('~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa') }} root@{{ source_os_diff_config_ip }}"
+    regex: "ssh -F ssh.config"
+    value: "ssh_cmd=ssh -F {{ os_diff_dir }}/ssh.config"
 
 - name: Set Tripleo config path in os-diff.cfg
   ansible.builtin.lineinfile:
+    firstmatch: true
     path: "{{ os_diff_dir }}/os-diff.cfg"
     regex: "local_config_path=/tmp/"
-    value: "local_config_path={{ os_diff_data_dir }}/tripleo"
+    value: "local_config_path={{ os_diff_data_dir }}"
 
 - name: Set OCP config path in os-diff.cfg
   ansible.builtin.lineinfile:
     path: "{{ os_diff_dir }}/os-diff.cfg"
     regex: "ocp_local_config_path=/tmp/ocp"
-    value: "ocp_local_config_path={{ os_diff_data_dir }}/ocp"
+    value: "ocp_local_config_path={{ os_diff_data_dir }}"
 
 - name: Set config path in os-diff.cfg
   ansible.builtin.lineinfile:
     path: "{{ os_diff_dir }}/os-diff.cfg"
     regex: "local_config_dir=/tmp/"
     value: "local_config_dir={{ os_diff_data_dir }}"
 
+# Workaround manual install
+- name: Create os-diff config dir in /etc
+  become: true
+  ansible.builtin.file:
+    path: /etc/os-diff
```

Comment:
Can we make it without sudo? E.g. provide an alternative config file path via a command option or an environment variable?